app/Providers/RouteServiceProvider.php

```diff
@@ -33,11 +33,21 @@ public function boot()
                 ->prefix('api')
                 ->group(base_path('routes/api.php'));
 
-//            Route::prefix('auth')
-//                 ->group(base_path('routes/qr-menu-routes/web-auth.php'));
-
             Route::middleware('web')
+                ->prefix('public')
```

Comment:
public here is not required.. simply left without prefix... it will result in https://www.qr-menu.com/ (home of the proj)..